p2p/pre2p/handlers.go

```diff
@@ -0,0 +1,37 @@
+package pre2p
+
+import (
+	"io/ioutil"
+	"log"
+	"net"
+
+	"github.com/pokt-network/pocket/shared/types"
+
+	"google.golang.org/protobuf/proto"
+)
+
+func (m *p2pModule) handleNetworkMessage(conn net.Conn) {
```

Comment:
This code and all pre2p code feels out of scope? Can you justify this code's relevance? Is it the result of a merge / rebase? I will still review it, but this might be important